File: backend/Entities/Models/DocumentoFilialGrupo.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class DocumentoFilialGrupo : BaseEntity
{
    public Guid Id { get; set; }

    public Guid EnderecoServicoId { get; set; }

    public int FilialId { get; set; }

    public int? FilialAtualId { get; set; }

    public int? FilialDestinoId { get; set; }

    public Guid? FilialLastMileGrupoItemId { get; set; }

    public Guid? TransportadorLastMileGrupoItemId { get; set; }

    public virtual Endereco EnderecoServico { get; set; } = null!;

    public virtual Documento1 IdNavigation { get; set; } = null!;
}

File: frontend-with-menus/src/app/Business/Models/documentoFilialGrupo.ts
import { Documento1 } from "./documento1";
import { Endereco } from "./endereco";

export interface DocumentoFilialGrupo  {
    id: string;
    enderecoServicoId: string;
    filialId: number;
    filialAtualId: number | null;
    filialDestinoId: number | null;
    filialLastMileGrupoItemId: string | null;
    transportadorLastMileGrupoItemId: string | null;
    enderecoServico: Endereco;
    idNavigation: Documento1;
}